launch.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+if [ $# -lt 1 ]; then
+  echo "Launch any of the following examples with 'npm start <example>'"
+  echo "-----------------------------------------------------------------------------"
+  ls ./examples/ --width 80 --color=never
+  echo "-----------------------------------------------------------------------------"
+  exit
+fi
+npm install --prefix examples/$1
+if [ "$2" == '--dev' ]; then # ./launch.sh <example> --dev
+  npm install
+  npm run build
+  rm examples/$1/node_modules/tabris -rf
+  rm examples/$1/node_modules/tabris-decorators -rf
+  npm install tabris@nightly --no-save --prefix examples/$1
```

Comment:
Do we want to install tabris@nightly? See my other comments.

Reply:
This is only if you use the --dev parameter, in which case yes, it is intended. Otherwise it uses the version specified in the example's package.json